build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -48,22 +50,60 @@ private string GetCrossgenPathForVersion()
 
         private string GetLibCLRJitPathForVersion()
         {
-            var coreclrRid = GetCoreCLRRid();
-            var crossgenPackagePath = GetCrossGenPackagePathForVersion();
+            var jitRid = GetCoreCLRRid();
+            var jitPackagePath = GetJitPackagePathForVersion();
 
-            if (crossgenPackagePath == null)
+            if (jitPackagePath == null)
             {
                 return null;
             }
 
             return Path.Combine(
-                crossgenPackagePath,
+                jitPackagePath,
                 "runtimes",
-                coreclrRid,
+                jitRid,
                 "native",
                 $"{Constants.DynamicLibPrefix}clrjit{Constants.DynamicLibSuffix}");
         }
 
+        private string GetJitPackagePathForVersion()
+        {
+            string jitRid = GetCoreCLRRid();
+
+            if (jitRid == null)
+            {
+                return null;
+            }
+
+            string packageId = $"runtime.{jitRid}.Microsoft.NETCore.Jit";
+
+            return Path.Combine(
+                Dirs.NuGetPackages,
+                packageId,
+                _jitVersion);
+        }
+
+        private string GetCoreLibsDirForVersion()
```

Comment:
Can this name match the package name better?  `CoreLibs` doesn't really say `Runtime.CoreCLR` to me.